site/gatsby-site/src/components/forms/IncidentReportForm.js

```diff
@@ -168,8 +168,13 @@ const IncidentReportForm = ({ incident, onUpdate, onSubmit }) => {
         };
       });
     } catch (e) {
+      const message =
+        e.message == 'Parser error'
+          ? `Error fetching news from ${newsUrl}, you'll have to enter the text manually.`
```

Comment:
`Error fetching news. Scraping was blocked by ${newsUrl}, Please enter the text manually.`